cmd/meroxa/turbine/ruby/deploy.go

```diff
@@ -92,10 +92,15 @@ func (t *turbineRbCLI) GetResources(ctx context.Context, appName string) ([]util
 	}
 
 	var resources []utils.ApplicationResource
-	for _, r := range resp.Resources {
-		resources = append(resources, utils.ApplicationResource{
-			Name: r.Name,
-		})
+	for _, r := range resp.GetResources() {
+		for _, s := range r.GetDescription() {
```

Comment:
If api just wants a list of resources, maybe the way core should store them is in just a list format. 
for loops inside for loops scare me